source/FAST/Visualization/WindowWidget.cpp

```diff
@@ -17,6 +20,28 @@ void WindowWidget::keyPressEvent(QKeyEvent *event) {
         view->keyPressEvent(event);
 }
 
+void WindowWidget::dragEnterEvent(QDragEnterEvent *event) {
+    Reporter::info() << "\nDrag event..." << Reporter::end();
+    if (event->mimeData()->hasUrls()) {
+        event->acceptProposedAction();
+    }
+}
+
+
+void WindowWidget::dropEvent(QDropEvent *event) {
+    std::cout << "\nDrop event..." << std::endl;
+    QList<QString> fileNames;
+    foreach (const QUrl &url, event->mimeData()->urls()) {
+        QString fileName = url.toLocalFile();
+        fileNames.push_back(fileName);
+        Reporter::info() << "Dropped file:" << fileName.toStdString() << Reporter::end();
+    }
+    foreach(QString curr, fileNames) {
```

Comment:
This extra foreach loop doesn't seem to have a purpose

Reply:
Yeah, I was using it for debugging, and forgot to remove it. Thanks!